holmes/plugins/toolsets/internet/notion.py

```diff
@@ -125,3 +126,12 @@ def __init__(self):
             ],
             is_default=False,
         )
+
+    def prerequisites_callable(self, config: Dict[str, Any]) -> Tuple[bool, str]:
+        if not config or not config.get("additional_headers", {}):
+            return (
+                False,
+                "Notion toolset is misconfigured. Authorization header is required.",
+            )
+        self.additional_headers = config.get("additional_headers", {})
```

Comment:
We need to also the docs?

Reply:
Not sure what you mean

Reply:
We need to also update the docs on robusta?